src/main/java/org/broadinstitute/hellbender/tools/funcotator/dataSources/DataSourceUtils.java

```diff
@@ -371,6 +410,147 @@ private static Properties readConfigFileProperties(final Path configFilePath) {
         return configProperties;
     }
 
+    /**
+     * Logs the meta data about the data sources from the given {@code dataSourcesPath}.
+     * We assume the data sources path is OK in the case that the version information cannot be read because the
+     * user can create their own data sources directory, which may not contain the metadata we seek.
+     * @param dataSourcesPath {@link Path} to a Data Sources directory to check.
+     * @return {@code True} if the given {@code dataSourcesPath} is equal to or newer than the minumum version, or if the version is unreadable.  {@code False} otherwise.
+     */
+    private static boolean logDataSourcesInfo(final Path dataSourcesPath) {
+
+        boolean dataSourcesPathIsAcceptable = true;
+
+        final Path readmePath = dataSourcesPath.resolve(IOUtils.getPath(README_FILE_NAME));
+
+        String version = null;
+
+        if ( Files.exists(readmePath) && Files.isRegularFile(readmePath) && Files.isReadable(readmePath) ) {
+
+            try ( final BufferedReader reader = Files.newBufferedReader(readmePath) ) {
+
+                Integer versionMajor   = null;
+                Integer versionMinor   = null;
+                Integer versionYear    = null;
+                Integer versionMonth   = null;
+                Integer versionDay     = null;
+                String source          = null;
+                String alternateSource = null;
+
+                // Get the info from our README file:
+                String line = reader.readLine();
+                while ((line != null) && ((version == null) || (source == null) || (alternateSource == null))) {
+
+                    if (version == null && line.startsWith(README_VERSION_LINE_START)) {
+                        final Matcher m = VERSION_PATTERN.matcher(line);
+                        if ( m.matches() ) {
+                            versionMajor  = Integer.valueOf(m.group(1));
+                            versionMinor  = Integer.valueOf(m.group(2));
+                            versionYear   = Integer.valueOf(m.group(3));
+                            versionMonth  = Integer.valueOf(m.group(4));
+                            versionDay    = Integer.valueOf(m.group(5));
+
+                            version = versionMajor + "." + versionMinor + "." + versionYear + "" + versionMonth + "" + versionDay;
+                        }
+                        else {
+                            logger.warn("README file has improperly formatted version string: " + line);
```

Comment:
README files typically have minimal to zero formatting.  This would indicate that we have a stricter format.  Is there a good reason not to merge with the config file?

Reply:
@LeeTL1220 There is no config file for a data sources directory - only a README file.

I guess we can call it a config file, but then no one is likely to read it and it provides information to anyone who would download / use the data sources.

Reply:
Leave w/ README.  Again, I thought this was being run for each datasource.  Not once for all datasources.